torch/_guards.py

```diff
@@ -119,6 +119,7 @@ class Guard:
     name: str
     source: GuardSource
     create_fn: Callable[[GuardBuilderBase, "Guard"], None]
+    origin: Optional["Source"] = None
```

Comment:
why not at the end 🤔

Reply:
Does this have to be optional. Easier to audit if not

Reply:
Sadly yes